src/harness/fourslash.ts

```diff
@@ -4968,8 +4951,20 @@ namespace FourSlashInterface {
             "let",
             "package",
             "yield",
+            "any",
             "async",
             "await",
+            "boolean",

```

Comment:
Why? I don't think we should be showing these here

Reply:
I think you want to check if 
`let m = new Map<string/**/` is  isPossiblyTypeArgumentPosition and add type keywords + function value keywords

Reply:
As far as I can tell, there's no more precise context than "inside a function" for completions. I think if we show things like `class, var, interface` there then type keywords make sense too.

Reply:
don't think so. `class, var, interface ` can really be used in the function context to start local declarations? why would you want types?

Reply:
`filterGlobalCompletion` has code that looks like it tries to do this, but it relies exclusively on `isTypeOnlyCompletion`. Is that a good place to add a check for isPossiblyTypeArgumentPosition, do you think?